drizzlepac/devutils/json_splitter.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
```

Comment:
Does this file need to be provided to the end-user?  If not, it should be removed from this PR.